### PR TITLE
provide `angles` `innerangles` for Ngons, with docstrings, tests

### PR DESCRIPTION
**Pull request by JeffreySarnoff**

### CODE REVIEW

**Comment by JeffreySarnoff** [comment]
The docs for `angles` says "Positive angles represent a CCW rotation"

`t = Triangle(Point2(0,0), Point2(1,0), Point2(0,1))` does go counterclockwise,
but the angles are negative -- why?

This is not a function of the PR (afaik).  Is this the way it should be?

File: src/polytopes/ngon.jl [hunk 1]
```diff
@@ -65,6 +65,20 @@ chains(ngon::Ngon{N}) where {N} = [Chain(ngon.vertices[[1:N; 1]])]
 
 Base.unique!(ngon::Ngon) = ngon
 
+"""
+    angles(ngon)
+
+Return the angles of the `ngon` in vertex order.
```
Comment (juliohm) [inline src/polytopes/ngon.jl]:
```suggestion
Return the angles of the boundary of the `ngon`.

See also [`Chain`](@ref).
```

File: src/polytopes/ngon.jl [hunk 2]
```diff
@@ -65,6 +65,22 @@ chains(ngon::Ngon{N}) where {N} = [Chain(ngon.vertices[[1:N; 1]])]
...
+Return the angles of the boundary of the `ngon`.
+
+See also [`Chain`](@ref).
+"""
+angles(ngon::Ngon) = angles(boundary(ngon))
+
+"""
+    innerangles(ngon)
+
+Return inner angles of the `ngon` in vertex order.
```
Comment (juliohm) [inline src/polytopes/ngon.jl]:
```suggestion
Return inner angles of the boundary of the `ngon`.

See also [`Chain`](@ref).
```

File: test/angles.jl [hunk 3]
```diff
@@ -16,4 +16,9 @@
   @test ∠(P3(0,1,0),P3(0,0,0),P3(0,0,1)) ≈ T(π/2)
   @test ∠(P3(0,0,1),P3(0,0,0),P3(1,0,0)) ≈ T(π/2)
   @test ∠(P3(0,0,1),P3(0,0,0),P3(0,1,0)) ≈ T(π/2)
+  
+  # Ngon
+  t = Triangle(Point2(0,0), Point2(1,0), Point2(0,1))
+  @test all(isapprox.(rad2deg.(angles(t)), [-90.0, -45.0, -45.0], atol=8*eps(45.0)))
+  @test all(isapprox.(rad2deg.(innerangles(t)), [90.0, 45.0, 45.0], atol=8*eps(45.0)))
```
Comment (juliohm) [inline test/angles.jl]:
```suggestion
  t = Triangle(P2(0,0), P2(1,0), P2(0,1))
  @test all(isapprox.(rad2deg.(angles(t)), T[-90, -45, -45], atol=atol(T)))
  @test all(isapprox.(rad2deg.(innerangles(t)), T[90, 45, 45], atol=atol(T)))     
```

File: test/angles.jl [hunk 4]
```diff
@@ -16,4 +16,9 @@
   @test ∠(P3(0,1,0),P3(0,0,0),P3(0,0,1)) ≈ T(π/2)
   @test ∠(P3(0,0,1),P3(0,0,0),P3(1,0,0)) ≈ T(π/2)
   @test ∠(P3(0,0,1),P3(0,0,0),P3(0,1,0)) ≈ T(π/2)
+  
+  # Ngon
+  t = Triangle(P2(0,0), P2(1,0), P2(0,1))
+  @test all(isapprox.(rad2deg.(angles(t)), T[-90, -45, -45], atol=atol(T)))
+  @test all(isapprox.(rad2deg.(innerangles(t)), T[90, 45, 45], atol=atol(T)))
```
Comment (juliohm) [inline test/angles.jl]: Can you please move these tests to the Ngon section in the test suite?

File: test/angles.jl [hunk 5]
```diff
@@ -15,5 +15,5 @@
   @test ∠(P3(0,1,0),P3(0,0,0),P3(1,0,0)) ≈ T(π/2)
   @test ∠(P3(0,1,0),P3(0,0,0),P3(0,0,1)) ≈ T(π/2)
   @test ∠(P3(0,0,1),P3(0,0,0),P3(1,0,0)) ≈ T(π/2)
-  @test ∠(P3(0,0,1),P3(0,0,0),P3(0,1,0)) ≈ T(π/2)
+  @test ∠(P3(0,0,1),P3(0,0,0),P3(0,1,0)) ≈ T(π/2)
```
Comment (juliohm) [inline test/angles.jl]:
```suggestion
  @test ∠(P3(0,0,1),P3(0,0,0),P3(0,1,0)) ≈ T(π/2)
```

**Pull request merged**